services/order.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Order, Ticket, User
+from django.db import transaction
+from datetime import datetime
+
+
+def create_order(
+        tickets: list,
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)

```

Comment:
Consider handling the `DoesNotExist` exception that may be raised by `User.objects.get` to provide a more user-friendly error message or behavior.